docs/data_dictionaries/usage_warnings.rst

```diff
@@ -0,0 +1,103 @@
+=====================================================================================================
+PUDL Usage Warnings
+=====================================================================================================
```

Comment:
Most text in this file is the same as the usage warning text in metadata/warnings.py. I'll flag the two places I added more info.